tests/Unit/Evolution/Systems/Cce/Test_BoundaryData.cpp

```diff
@@ -25,6 +26,10 @@ namespace Cce {
 namespace {
 
 void pypp_test_worldtube_computation_steps() noexcept {
+  // This test triggers some kind of UB under python 3.  See #1867.
```

Comment:
This might be fixed with #1889. There is actual UB in SpECTRE code that is fixed and will hopefully deal with this.